namespace cpp Ruyi.SDK.Constants
namespace csharp Ruyi.SDK.Constants
namespace java Ruyi.SDK.Constants
namespace netcore Ruyi.SDK.Constants
namespace rs Ruyi.SDK.Constants

/** @low_latency_socket_port_desc */
const i32 low_latency_socket_port = 11290

/** @high_latency_socket_port_desc */
const i32 high_latency_socket_port = 11390

/** @layer0_broker_address_desc */
const string layer0_broker_address = "tcp://{addr}:5555"

/** @layer0_publisher_in_uri_desc */
const string layer0_publisher_in_uri = "tcp://{addr}:5567"

/** @layer0_publisher_out_uri_desc */
const string layer0_publisher_out_uri = "tcp://{addr}:5568"

/** @setting_config_folder_desc */
const string setting_config_folder = "resources/configs/"

/** @setting_system_config_desc */
const string setting_system_config = "resources/configs/systemsetting"

/** @setting_system_user_config_desc */
const string setting_system_user_config = "resources/configs/usersetting"

/** @system_setting_version_desc */
const string system_setting_version = "1.0.0.1"

/** @layer0_debugger_channel_desc */
const string layer0_debugger_channel = "layer0_debugger_channel"

/** @trc_test_channel_desc */
const string trc_test_channel = "trc_test_channel"

/** @broker_playback_message_desc */
const string broker_playback_message = "mmi.developer.playback"

/** @HDD0_DRIVER_TAG_desc */
const string HDD0_DRIVER_TAG = "/<hdd0>/"

/** @MEMCACHE_DRIVER_TAG_desc */
const string MEMCACHE_DRIVER_TAG = "/<memcache>/"

/** @HTTP_HDD_CACHE_DRIVER_TAG_desc */
const string HTTP_HDD_CACHE_DRIVER_TAG = "/<httphddcache>/"

/** @HTTP_MEM_CACHE_DRIVER_TAG_desc */
const string HTTP_MEM_CACHE_DRIVER_TAG = "/<httpmemcache>/"

/** @RESOURCES_DRIVER_TAG_desc */
const string RESOURCES_DRIVER_TAG = "/<resources>/"

/** @MEDIA_DRIVER_TAG_desc */
const string MEDIA_DRIVER_TAG = "/<media>/"

/** @DOWNLOAD_DRIVER_TAG_desc */
const string DOWNLOAD_DRIVER_TAG = "/<download>/"

/** @DATA_DRIVER_TAG_desc */
const string DATA_DRIVER_TAG = "/<data>/"


